cns/service/main.go

```diff
@@ -1734,7 +1715,6 @@ func createOrUpdateNodeInfoCRD(ctx context.Context, restConfig *rest.Config, nod
 		},
 		Spec: mtv1alpha1.NodeInfoSpec{
 			VMUniqueID: vmUniqueID,
-			HomeAZ:     homeAZ,
```

Comment:
Will this revert break existing clusters where the new Node Spec is already deployed?

Reply:
make file requires the push of this change